features/mbedtls/VERSION.txt

```diff
@@ -1 +1 @@
-mbedtls-2.19.0
+mbedtls-2.17.0-1047-gb2ab86ca1594
```

Comment:
Is that correct ? anyway will be updated here once the release is ready, won't be?

Reply:
That's the auto-version labeler not being able to find the tag. We have the tag now, and so this is now fixed.